js/src/client.ts

```diff
@@ -1892,7 +1893,14 @@ export class Client {
     {
       description,
       dataType,
-    }: { description?: string; dataType?: DataType } = {}
+      inputsSchema,
```

Comment:
do we want to add to the update method as well?

Reply:
eh I can add later? I only need this for passing integration tests